jaeger-jaxrs2/build.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'jacoco'
+apply plugin: 'com.github.kt3k.coveralls'
+
+description = 'Instrumentation library for jaeger-jaxrs 2.0'
+
+dependencies {
+    compile group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.0'
+    compile group: 'com.uber.jaeger', name: 'jaeger-core', version: version
+
+    testCompile group: 'org.glassfish.jersey.containers', name: 'jersey-container-grizzly2-http', version: jerseyVersion
+    testCompile group: 'org.glassfish.jersey.containers', name: 'jersey-container-servlet', version: jerseyVersion
+    testCompile group: 'org.glassfish.jersey.core', name: 'jersey-client', version: jerseyVersion
+    testCompile group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: jerseyVersion
+    testCompile group: 'org.glassfish.jersey.media', name: 'jersey-media-moxy', version: jerseyVersion
+    testCompile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.7.4'
+}
+
+jacocoTestReport {
```

Comment:
to clarify, my question was - can we have it just once, instead of in every module?